debugger.jpda.projects/src/org/netbeans/modules/debugger/jpda/projects/IntroduceClass.java

```diff
@@ -124,6 +124,7 @@ boolean computeIntroduceMethod(TreePathHandle h, CompilationInfo info, TreePath
                 lsBegin -= codeOffset;
                 this.snippetCode = new StringBuilder(this.snippetCode)
                         .insert((int) lsBegin, "return ")
+                        .append(';')
```

Comment:
This is only important for vanilla javac, nb-javac's error recovery is able to handle the missing semi-colon. But having the semicolon should not do any harm.